### PR TITLE
Add basic resources page

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add resources page (renamed from tools)
- get tags from metadata, and from github if repo is provided in metadata
- update resource thumbnails
- make css for resource cards of different sizes
- on research page, if error loading image, fallback to default thumbnail
- add icons to page titles
- various small icon, css, and include changes
- fix bug where directory-specific css and js files were being included from unintended directories
- try to preload home page background image so maybe it loads before page is rendered and users don't see a flash of nothing?
- move Olivia Cheng to alum

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- title: Childhood Cancer Data Lab
+  thumbnail: ccdl.png
+  url: https://www.ccdatalab.org/projects
+  description: Turning researchers who fight childhood cancer into data wizards.
+  affiliated: true
+
+- title: Hetio
+  thumbnail: https://het.io/share-thumbnail.png
+  url: https://github.com/
```
Comment (cgreene) [inline _data/tools.yml:52]:
```suggestion
  url: https://github.com/hetio
```

**Comment by cgreene** [comment]
I am still thinking about what we want to term this, but I wonder if "Software" is a better choice?

Also, what do you think about having legacy software? nano-dissection, giant, imp, etc are nifty software but a bit long in the tooth (i.e., like more than 5 years old now). Should we separate that out and use this page to highlight new software?

I'm also imagining that if we do software, something like greenelab/website-rootstock fits in software, but I'm not sure it's a "tool" per se.

**Comment by vincerubinetti** [comment]
Software would be okay. I just named it tools to be a little bit more general, like if there was a useful thing that wasn't necessarily software, like a dataset or something else I can't think of right now. "Tools" meaning just something generally useful.

We can have a legacy flag, similar to the alum flag in team bios, that will make them get listed in their own category, maybe with smaller thumbnails and less text.

**Comment by vincerubinetti** [comment]
I'd say we merge this for now as a basic page and add the legacy flag when we come back and add more tools and thumbnails.

**Comment by cgreene** [comment]
What about "Resources" instead of tools? I feel like there is software that's not really a tool. I also feel like a dataset wouldn't be a tool. On the other hand, both would be resources. Especially if this becomes a rootstock, "resources" is likely to work much more broadly.

Also, what if we flipped the idea from "Legacy" to "Recent Highlights" - the recent highlights would be displayed larger / more prominently at the top, and I agree the others would get smaller thumbnails and less text.

This is looking great!

**Comment by vincerubinetti** [comment]
@cgreene take another look

**Review by cgreene — APPROVED** [review]
OMG yes! The SVG thing is a great example of a resource! 🤩

**Pull request merged**